features/sync_features.py

```diff
@@ -21,6 +21,7 @@
     HOLD_PENALTY            = 0x40,
     TIMEOUT_PENALTY         = 0x41,
     DELAY_PENALTY           = 0x42,
+    FIXATION_PENALTY        = 0x43,
```

Comment:
because we changed the event dictionary, i'd like to bump the sync protocol version number, below on line 67 from 13 to 14